README.md

```diff
@@ -296,6 +296,12 @@ spec:
         - --node=""
 ```
 
+| KSM --node parameter | FieldSelector |
```

Comment:
Can we have a human readable column as well that explains what this setting does?

Reply:
Updated